traffic_ops/app/lib/API/User.pm

```diff
@@ -281,11 +284,14 @@ sub current {
 
 	if ( &is_ldap($self) ) {
 		my $role = $self->db->resultset('Role')->search( { name => "read-only" } )->get_column('id')->single;
+		my $user_tenant_id = $self->current_user_tenant();
+		my $user_tenant = defined($user_tenant_id) ? $self->db->resultset('Tenant')->search( { id => $user_tenant_id } )->get_column('name')->single : "N/A";
```

Comment:
Same as the question above on prefetch. In this case the prefetch for "role" does not exist either, but the API still works without it and after my change it works without prefetch for tenant table. So I am missing something on how it works and when prefetch must be used.